cmd/node/node.go

```diff
@@ -306,7 +306,7 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId, swarm service.Service
 	msh := mesh.NewMesh(mdb, atxdb, app.Config.REWARD, trtl, processor, lg.WithName("mesh")) //todo: what to do with the logger?
 
 	conf := sync.Configuration{SyncInterval: 1 * time.Second, Concurrency: 4, LayerSize: int(layerSize), RequestTimeout: 100 * time.Millisecond}
-	syncer := sync.NewSync(swarm, msh, blockValidator, conf, clock.Subscribe(), lg.WithName("sync"))
+	syncer := sync.NewSync(swarm, msh, blockValidator, sync.TxValidatorMock{}, conf, clock.Subscribe(), lg.WithName("sync"))
```

Comment:
why are we using mock ?

Reply:
we have no tx validation right now and its not part of this pr